README.md

```diff
@@ -1,17 +1,29 @@
 # node-red-contrib-socketio-client
 ---
+## Latest Features
+ - Removed Conflicts with node-red-contrib-socketio-extended for server integrations.
+ - Added two listener nodes listener in and listener out for use with callback.
+ - Colour coded call back nodes.
 
 ## Nodes
 
 1. Socket.IO Connector
 2. Socket.IO Listener
+3. Socket.IO Emitter
+4. Socket.IO Emitter with Callback
+5. Socket.IO Listener in, for use with callback listener out
+6. Socket.IO Listener out, for use with callback listener in
 
 ## How to use
+Original Listener:
 
 > Socket.IO Connector -> Socket.IO Listener -> Payload
-
 ![How to use](https://raw.githubusercontent.com/isaacvitor/generalcontent/master/node-red-contrib-socketio-client/nodered_socketio_ex01.png "How to use")
 
+Listener with callback:
+
+> Socket.IO Connector -> Socket.IO Listener In -> Payload -> Socket.IO Listener Out
+![How to use](https://raw.githubusercontent.com/nateainsworth/Git-docs-images/master/node-red-contrib-socketio-client/listener%20with%20callback.png "How to use")
```

Comment:
Could you change this?

Reply:
Sorry, which bit do you want me to remove the images?

Reply:
this image is important... just to change the URL upload to my repo, please

Reply:
All done I've pointed it to your repo, but you'll want to upload the image to your repo download it from here https://github.com/nateainsworth/Git-docs-images/blob/master/node-red-contrib-socketio-client/listener%20with%20callback.png

As I couldn't see the image in your repo https://github.com/isaacvitor/generalcontent/tree/master/node-red-contrib-socketio-client

Reply:
I didn't upload 👎 sorry